modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -3482,6 +3482,71 @@ export const console: ConsoleNS = {
                         }
                     }
                 },
+                resident: {
+                    provisioning: {
+                        outbound: {
+                            actions: {
+                                addIdp: "නව කොමිෂන් සභා"
```

Comment:
Ref: https://translate.google.com/?sl=en&tl=si&text=New%20Provisioner&op=translate&hl=en

```suggestion
                                addIdp: "නව සැපයුම්කරු"
```